### DISCUSSION TITLE
[Feature] Delete, Merge Faces Together & Ability to Crop Photos.

### DISCUSSION THREAD

**Discussion by Awsomegameplay123** [immich-app/immich#3777] [Feature Request]
### The feature

It would help if we could hide/delete unwanted faces of random people as people in the background and on screenshots tend to be saved as a face. Also the ability to merge faces together would also help if there are 2 faces of the same person. Lastly, maybe ability to crop photos Atleast?

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
We can hide and merge faces already

**Reply by Awsomegameplay123** [reply]
How can you do it on mobile?

**Reply by JvdMaat** [reply]
Is there a way to unmerge?
Or selectively merge?
I have a case where there are two pictures, one containing a family relative, the other has no real face in it. However it is recognizing the family relative (in the first picture (though not as the other pictures of said relative) and thinks it's in the second. Can we selectively add just the one picture face match to merge?
And as a huge follow-up, is there a way to tell the AI that something is not a face, or to exclude that specific result?
And to keep building on that. Since you can set the face feature photo... Can there be a toggle to just show rectangles around all the discovered faces? Makes matching and finding a lot easier.
And lastly (for now), could you add hide face to the menu when looking at a person? (rather than having to click into the face, see it's nobody, go back to the full face gallery, attempt to find that specific face again, and then hide it from that view).
Happy to log Issues for any of these if they aren't on the roadmap and make sense.
